ansible/playbooks/jenkins/worker/upgrade-jar.yml

```diff
@@ -10,8 +10,32 @@
     - name: download slave.jar
       get_url:
         url: "{{ jenkins_worker_jar }}"
-        dest: /home/{{ server_user }}/slave.jar
+        dest: '{{ home }}/{{ server_user }}/slave.jar'
         mode: 0644
+        force: yes
 
-    - name: restart jenkins
+    - name: Kill Jenkins process - AIX
+      shell: ps awwx | grep jenkins | grep -v grep | awk '{ print $1 }' | xargs kill
+      when: os|startswith("aix")
```

Comment:
Maybe we should implement `stop` in the rc.d script? https://github.com/nodejs/build/blob/2a48e7f4c4e91bff403e346885ffc20150b1e5a4/ansible/roles/jenkins-worker/templates/aix.rc2.j2#L25-L26

Reply:
hmm yeah that should be something we should look into, I never realised it was even defined in the script.